bump-api.py

```diff
@@ -27,9 +27,10 @@ def version_path():
     file_path = os.path.join(tmpdir, file_name)
     return file_path
 
-
 def version_inc(path, msg):
-    return check_output("bump --filename {} {}".format(path, msg), stderr=STDOUT, shell=True)
+    command = ["bump", "--filename", path, msg]
```

Comment:
This addresses unsanitised input from the HTTP request body flows into subprocess.check_output, where it is used as a shell command